spec/controllers/api/v1/players_controller_spec.rb

```diff
@@ -44,4 +44,14 @@
       expect(returned_json[1]['weight']).to eq '240'
     end
   end
+  describe 'GET#show' do
```

Comment:
this controller action is returning more than just a player, so your test should be accounting for everything the controller returns